core/cloudflow-blueprint/src/main/scala/cloudflow/blueprint/deployment/ApplicationDescriptor.scala

```diff
@@ -54,6 +54,7 @@ object ApplicationDescriptor {
   def apply(
       appId: String,
       appVersion: String,
+      img: String,
```

Comment:
very minor `img` -> `image`?

Reply:
@RayRoestenburg What is the Scala syntax of a case class copy when both the names are same ? Will `deployment.copy(image = image)` work ?

Reply:
yes.

Reply:
done ..